packages/react/src/QuestionnaireBuilder/QuestionnaireBuilder.tsx

```diff
@@ -371,16 +378,72 @@ function ItemBuilder<T extends Questionnaire | QuestionnaireItem>(props: ItemBui
 }
 
 interface AnswerBuilderProps {
-  options?: QuestionnaireItemAnswerOption[];
-  onChange: (newOptions: QuestionnaireItemAnswerOption[]) => void;
+  item: QuestionnaireItem;
+  onChange: (item: QuestionnaireItem) => void;
 }
 
 function AnswerBuilder(props: AnswerBuilderProps): JSX.Element {
   const property = globalSchema.types['QuestionnaireItemAnswerOption'].properties['value[x]'];
-  const options = props.options ?? [];
+  const options = props.item.answerOption ?? [];
+  return (
+    <div>
+      {props.item.answerValueSet !== undefined ? (
```

Comment:
When pressing 'Add value set', it'll create one with a value of '' so I have to do this rather than !!props.item.answerValueSet